SPECS/mstflint/mstflint.spec

```diff
@@ -23,15 +23,35 @@ BuildRequires:  libxml2-devel
 BuildRequires:  make
 BuildRequires:  openssl-devel
 BuildRequires:  zlib-devel
+BuildRequires:  kernel-devel
```

Comment:
The list of BuildRequires is suppossed to be sorted alphabetically to make it easier to maintain the code. The newly added lines violate this convention, so please rearrange them accordingly.